fluent/fluent_test.go

```diff
@@ -248,6 +268,51 @@ func TestAsyncConnect(t *testing.T) {
 	}
 }
 
+func Test_PostWithTime(t *testing.T) {
+	f, err := New(Config{
+		FluentPort:    6666,
+		AsyncConnect:  false,
+		MarshalAsJSON: true, // easy to check equality
+	})
+	if err != nil {
+		t.Error(err)
+		return
+	}
+
+	var testData = []struct {
+		in  map[string]string
+		out string
+	}{
+		{
+			map[string]string{"foo": "bar"},
+			"[\"tag_name\",1482493046,{\"foo\":\"bar\"},null]",
+		},
+		{
+			map[string]string{"fuga": "bar", "hoge": "fuga"},
+			"[\"tag_name\",1482493046,{\"fuga\":\"bar\",\"hoge\":\"fuga\"},null]",
```

Comment:
Yes, JSON should be sorted by keys. 

- ref. https://github.com/golang/go/commit/181000896e381f07e8f105eef2667d566729f6eb